lang/python/python-dbus-fast/Makefile

```
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2023
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-dbus-fast
+PKG_VERSION:=1.84.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=dbus_fast
+PKG_HASH:=62b00b85c5835bff1d7ab5b12d494e588d92612bedbd7ca86176861729b8e4bc
+
+PKG_MAINTAINER:=Quintin Hill <stuff@quintin.me.uk>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-dbus-fast
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=A faster version of dbus-next
+  URL:=https://github.com/Bluetooth-Devices/dbus-fast
+  DEPENDS:=+python3-light +python3-async-timeout +python3-asyncio \
+	+python3-distutils +python3-logging +python3-urllib +python3-xml
```

```suggestion
	+python3-logging +python3-urllib +python3-xml
```

distutils is only used in [build_ext.py](https://github.com/Bluetooth-Devices/dbus-fast/blob/v1.84.2/build_ext.py#L4), which should be only used during build/setup and not run-time.